infrastructure/oss/issue_html.go

```diff
@@ -114,7 +117,7 @@ type IntroducedThrough struct {
 }
 
 func getIntroducedThroughs(issue snyk.OssIssueData) []IntroducedThrough {
-	introducedThroughs := []IntroducedThrough{}
+	var introducedThroughs []IntroducedThrough
```

Comment:
nitpick: shouldn't it be `introducedThrough`?

Reply:
It's mean to be a list of `IntroducedThrough`  so that's why it's the extra `s`